conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalyticsIntegration.java

```diff
@@ -252,6 +262,16 @@ public void release(Boolean releaseConvivaSdk) {
         isSessionActive = false;
     }
 
+    /**
+     * Sends a stalled event during playback to Conviva's Player Insight. If no session is active it will NOT
+     * create one.
+     */
+    public void reportPlaybackStalled() {
```

Comment:
What is the use-case for adding this public API to manually report stalling? 🤔 And do we need a counter part to report 'end' stalling manually as well?

Reply:
While custom retry is happening, I needed to tell Conviva that buffering is happening.   Otherwise, Conviva would still think playback is still playing.